platform/android/export/export.cpp

```diff
@@ -47,6 +47,7 @@ void register_android_exporter() {
 	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::STRING, "export/android/debug_keystore", PROPERTY_HINT_GLOBAL_FILE, "*.keystore,*.jks"));
 	EDITOR_DEF("export/android/debug_keystore_user", "androiddebugkey");
 	EDITOR_DEF("export/android/debug_keystore_pass", "android");
+	EDITOR_DEF("export/android/debug_keystore_user_pass", "android");
```

Comment:
Why is this called "User Pass" instead of "Key Pass"?

Reply:
The documentation author used "user" for "--ks-key-alias", and to correspond with "user_password" for "--key-pass" .

Reply:
The Android export documentation does need to be changed. I personally feel that developers should not be forced to use the same password to sign the apk. This is wrong. For example, use the newly generated keystore file to sign for updating the same package with different names. I guess the APP signed by the keystore file cannot be updated and replaced, because the signature fingerprints are different, which is not conducive to the migration of Android projects migrated from other IDEs.
Moreover, the command parameter "--key-pass" was not added, which caused a signature error when godot ran to an Android device.
The password of the keystore is used as the alias password in order not to conflict with the author's intention as much as possible, although the author's intention is not understood.
I'm not very good at English, I hope you can edit the docs document.

Reply:
Use the password of the keystore as the default password when the key alias password is empty.
Sorry, I translated my text from the Internet, and there may be parts that are not easy to understand.